docs/advanced_usage.rst

```diff
@@ -602,6 +602,7 @@ and database modules without the need for changing your code::
 
   Usage: ddtrace-run <my_program>
 
+`--status`: This argument prints an easily readable tracer health check and configurations. It does not reflect configuration changes made during runtime.
```

Comment:
```suggestion
`--status`: This argument prints an easily readable tracer health check and configurations. It does not reflect configuration changes made during runtime (ex. using `tracer.configure()`)
```

Reply:
Maybe also worth adding docs to the quickstart page so users have an easy way to verify their install?

Reply:
+💯 💯 this would be great to the quickstart docs

install, set env variables, and then run this to verify everything is setup properly